src/java.desktop/share/classes/com/sun/imageio/plugins/tiff/TIFFFieldNode.java

```diff
@@ -122,9 +122,9 @@ private synchronized void initialize() {
                 child = new IIOMetadataNode("TIFFUndefined");
 
                 byte[] data = field.getAsBytes();
-                StringBuffer sb = new StringBuffer();
+                StringBuilder sb = new StringBuilder();
                 for (int i = 0; i < count; i++) {
-                    sb.append(Integer.toString(data[i] & 0xff));
+                    sb.append((data[i] & 0xff));
```

Comment:
Remove redundant parentheses

Reply:
fixed.
It's suspicious that IDEA wasn't able to remove them by itself. Reported https://youtrack.jetbrains.com/issue/IDEA-265548